helm-c-yasnippet.el

```diff
@@ -248,8 +252,13 @@ like `yas--current-key'"
                                       for name = (car dotlst)
                                       for template = (cdr dotlst)
                                       for key = (helm-yas-get-key-by-template template alist)
-                                      for name-inc-key = (concat "[" key "] " name)
-                                      collect `(,name-inc-key . ,template))))
+                                      collect `(,(format "[%s] %s"
```

Comment:
That would be OK with me. I personally prefer ``(format)``, but I can use ``(concat)`` too.

I had removed ``name-inc-key`` because it seemed a bit too much, but again I can put it back.

I'll update the PR ASAP.